extra/linux/io.alacritty.Alacritty.appdata.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2016-2019 Joe Wilm, The Alacritty Project Contributors -->
+<component type="desktop-application">
+  <id>io.alacritty.Alacritty</id>
+  <!-- Translators: The application name -->
+  <name>Alacritty</name>
+  <project_license>APACHE-2.0</project_license>
+  <metadata_license>APACHE-2.0</metadata_license>
+  <!-- Translators: The application's summary / tagline -->
+  <summary>A cross-platform, GPU enhanced terminal emulator</summary>
+  <description>
+    <p>Alacritty is a terminal emulator with a strong focus on simplicity and
+performance. With such a strong focus on performance, included features are
+carefully considered and you can always expect Alacritty to be blazingly fast.
+By making sane choices for defaults, Alacritty requires no additional setup.
+However, it does allow configuration of many aspects of the terminal.</p>
+    <p>Alacritty currently supports macOS, Linux, BSD, and Windows.</p>
```

Comment:
This last paragraph was intentionally removed for the manpage. Since this is a linux manifest, platform support is irrelevant.

So this should be removed.

Reply:
Ok. Will do.